### PR TITLE
add claim_signature_update and channel_update

### PR DESCRIPTION
**Pull request by jackrobison**

### CODE REVIEW

File: lbrynet/lbrynet_daemon/Daemon.py [hunk 1]
```diff
@@ -1694,6 +1694,39 @@ def jsonrpc_channel_new(self, channel_name, amount):
 
     @AuthJSONRPCServer.auth_required
     @defer.inlineCallbacks
+    @AuthJSONRPCServer.flags(revoke="-r")
+    def jsonrpc_channel_update(self, name, amount=None, revoke=False):
+        """
```
Comment (kaykurokawa) [inline lbrynet/lbrynet_daemon/Daemon.py]: name should be 'channel_name' to be consistent with jsonrpc_channel_new

**Comment by kaykurokawa** [comment]
I can't get channel_update command to work with revoke=False 

https://github.com/lbryio/lbryum/blob/master/lib/commands.py#L1605

Looks like you need to supply a val value to lbryum ? 

I'm going to force you to write tests for these on lbry-in-a-box. You can write the tests under _test_channels() in integration_testing.py

https://github.com/lbryio/lbry-in-a-box/blob/master/integration_testing.py#L539

You can make a PR with the tests on the lbry-in-a-box repo. To get travis tor run the integration test on the PR, change the line on .travis.yml that reads 'bash sync_to_master.sh' to 'bash sync_to_master.sh claim-signature-update master' which will use this branch for lbrynet and master for lbryum

**Comment by lyoshenka** [comment]
@jackrobison how much work is left here?

**Comment by eukreign** [comment]
Can this be closed and re-opened as a new issue in the new wallet?

**Comment by jackrobison** [comment]
@eukreign sure, if still relevant

**Pull request closed without merging**